code/game/objects/buckling.dm

```diff
@@ -40,9 +40,7 @@
 /atom/movable/proc/buckle_mob(mob/living/M, force = 0)
 	if(!buckled_mobs)
 		buckled_mobs = list()
-	if(!M.buckled_mobs)
```

Comment:
This didn't need Initialising, as if anything tried to buckle to ```M```, ```M ```would be ```src```, and thus get covered by the above.